models/demos/ttnn_resnet/tt/ttnn_functional_resnet50_new_conv_api.py

```diff
@@ -8,12 +8,37 @@
     is_grayskull,
     is_wormhole_b0,
     _nearest_y,
+    nearest_y,
     pad_and_fold_conv_activation_for_unity_stride,
 )
 from typing import List
 from loguru import logger
 from tests.ttnn.utils_for_testing import assert_with_pcc
 
+
+def get_core_grid_from_num_cores(num_cores: int, grid_rows: int, grid_cols: int):
```

Comment:
If the same function is defined in multiple places, maybe define it in a common location and import it.

Reply:
done